csharp/PhoneNumbers/BuildMetadataFromXml.cs

```diff
@@ -80,19 +83,19 @@ public class BuildMetadataFromXml
             bool isAlternateFormatsMetadata)
         {
             XDocument document;
-#if NET35
-            document = XDocument.Load(name);
-#else
-#if  NET40
+#if (NET35 || NET40)
             var asm = Assembly.GetExecutingAssembly();
 #else
             var asm = typeof(PhoneNumberUtil).GetTypeInfo().Assembly;
 #endif
             using (var input = asm.GetManifestResourceStream(name))
             {
+#if NET35
+		document = XDocument.Load(new XmlTextReader(input));
```

Comment:
nit but, can you fix the whitespace here?
```suggestion
                document = XDocument.Load(new XmlTextReader(input));
```

Reply:
Sure ;)

Please consider "squash and merge", because all those were trial and error!